lib/runner/extract-runnable-items.js

```diff
@@ -61,6 +61,49 @@ var sdk = require('postman-collection'),
         return matched;
     },
 
+    /**
+     * Finds items based on multiple ids or names provided.
+     *
+     * @param {ItemGroup} itemGroup - Composite list of Item or ItemGroup.
+     * @param {Object} entrypointSubset - Entry-points reference passed across multiple recursive calls.
+     * @param {Boolean} recursive - Flag used to decide whether to include nested items or not.
+     * @param {Array<String>} _accumulatedItems - Found Items or ItemGroups.
+     * @returns {Object}
+     */
+    findItemsOrGroups = function (itemGroup, entrypointSubset, recursive, _accumulatedItems) {
+        if (!itemGroup || !itemGroup.items) { return; }
+
+        !_accumulatedItems && (_accumulatedItems = []);
+
+        var idOrName;
```

Comment:
What does the variable name signify? After reading the code, it seems that it might change.